MainModule/Server/Commands/Fun.luau

```diff
@@ -3677,7 +3677,9 @@ return function(Vargs, env)
 						continue
 					end
 
-					if human and human.RigType == Enum.HumanoidRigType.R15 then
+					if human and char:isA("Model") then
```

Comment:
Just a minor nitpick you should convert isA to PascalCase instead of camelCase as :isA is deprecated by Roblox in favor of :IsA